modules/tests/src/test/scala/higherkindness/mu/rpc/protocol/Utils.scala

```diff
@@ -101,14 +88,7 @@ object Utils extends CommonUtils {
       def emptyAvroParamResponse: F[A]
       def emptyAvroWithSchemaParamResponse: F[A]
       def u(x: Int, y: Int): F[C]
-      def uws(x: Int, y: Int): F[C]
-      def uwe(a: A, err: String): F[C]
-      def ss(a: Int, b: Int): F[List[C]]
-      def ss192(a: Int, b: Int): F[List[C]]
-      def sswe(a: A, err: String): F[List[C]]
-      def cs(cList: List[C], bar: Int): F[D]
-      def bs(eList: List[E]): F[E]
-      def bsws(eList: List[E]): F[E]
```

Comment:
I removed these because the implementations were all using monix. I'm not sure if that's correct

Reply:
can we use fs2 and keep the tests or are they monix-specific?

Reply:
I don't know - I'll try digging into that today.

Reply:
👍